src/reference-manual/types.Rmd

```diff
@@ -573,12 +590,12 @@ packages R, MATLAB, BUGS, and JAGS.  General computer programming
 languages, on the other hand, such as C++ and Python, index arrays
 starting from zero.
 
-### Vectors {-}
+### Real vectors {-}
```

Comment:
Should the following "Row vectors" and "Matrices" subtitles also have "Real" prepended?

Reply:
I think we should leave "vector", "row vector" and "matrix" to mean regular ones.  Otherwise we'd have to clarify every single usage of "vector" in our doc!  So I went the other way and removed "real" from in front of "vector".